app/view/grid/Grid.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Base class for WFS-based data grids with filtering
+ * and Excel exports.
+ */
+Ext.define('CpsiMapview.view.grid.Grid', {
+    extend: 'Ext.grid.Panel',
+
+    requires: [
+        'Ext.toolbar.Paging',
+        'GeoExt.toolbar.WfsPaging',
+        'GeoExt.selection.FeatureModel',
+        'BasiGX.util.Layer',
+        'CpsiMapview.view.button.SpatialQueryButton'
+    ],
+
+    xtype: 'cmv_grid',
+    controller: 'cmv_grid',
+    viewModel: 'cmv_grid',
+
+    plugins: ['gridfilters'],
+    width: 1050,
+
+    bind: {
+        store: '{gridstore}'
+    },
+
+    viewConfig: {  //this config is passed to the view
+        loadMask: {
+            msg: 'Loading records' // TODO not sure why this isn't applied to the loadMask
+        }
```

Comment:
Could be achieved with `loadingText: 'Loading records'`, see https://docs.sencha.com/extjs/6.2.0/classic/Ext.view.Table.html#cfg-loadingText

Reply:
I've tried various properties such as `loadingText `, `loadMessage` in and out of the `viewConfig` with no luck. 
I can set it with `view.setLoading('Loading Records...');` in `onWfsStoreBeforeLoad` with no problems though.